corehq/apps/importer/tests.py

```diff
@@ -343,7 +386,8 @@ def testLocationOwner(self):
             ['', 'duplicate-location-name', '', duplicate_loc.name],
             ['', 'non-case-owning-name', '', improper_loc.name],
         ])
-        cases = {c.name: c for c in list(get_cases_in_domain(self.domain))}
+        case_ids = self.accessor.get_case_ids_in_domain()
+        cases = {c.name: c for c in list(self.accessor.get_cases(case_ids))}
```

Comment:
Because in general we would never want to fetch ALL cases in a domain in a single query (except in test code). The normal pattern is to select the case IDs and then use `accessor.iter_cases` to fetch them in batches.